urnx/types.go

```diff
@@ -0,0 +1,13 @@
+package urnx
+
+import "github.com/google/uuid"
+
+const prefix = "urn"
+const urnLength = 4
```

Comment:
Nit: We can make this less stutter-y like so:

```go
const (
        prefix    = "urn"
        urnLength = 4
)
```

Reply:
✅